admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -149,23 +300,31 @@ prevent session corruption when using Redis as your session handler: ::
 More information on configuration of phpredis session handler can be found on the
 `PhpRedis GitHub page <https://github.com/phpredis/phpredis>`_
 
-**Connecting to Redis over TLS:** ::
+Additional Redis installation help
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If your version of **Mint** or **Ubuntu** does not package the required version of 
+``php-redis``, then try `this Redis guide on Tech and Me
+<https://www.techandme.se/install-redis-cache-on-ubuntu-server-with-php-7-and-nextcloud/>`_
+for a complete Redis installation on Ubuntu 14.04 using PECL. 
+These instructions are adaptable for **any distro** that does not package the 
+supported version, or that does not package Redis at all, such as **SUSE Linux 
+Enterprise Server** and **Red Hat Enterprise Linux**.
+
+For PHP 7.0 and PHP 7.1 use Redis PHP module 3.1.x or later.
```

Comment:
I think we can spare us this section, for 7.4 is the minimum requirement at the moment.

Reply:
This is not my stuff. OF course it can be removed, but it has been there [before](ation/pull/9093/files#diff-48dc187eeb03371b5ad6886b33043bf060884809c1d51a3568c5d29e40e3bf13L249).

If you want me to remove it, please specify exactly which part (lines). But it is a shame, if the improvements of this PR are solely delayed due to old stuff. And honestly, I do not know enough about these technical details to be able to "write" the manual. I've only re-organized and added more examples and explanations ... :-)